easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0-bare.eb

```diff
@@ -32,4 +32,6 @@ dependencies = [
 
 install_pip = True
 
+runtest = 'test'
```

Comment:
@ocaisa I prefer doing this in the custom easyblock we have for Python (and probably only for sufficiently recent Python versions)